model/app.model.js

```diff
@@ -1,15 +1,21 @@
-const format = require('pg-format')
-const db = require('../db/connection.js')
-
+const format = require("pg-format");
+const db = require("../db/connection.js");
 
 const selectTopics = () => {
-    return db.query('SELECT * FROM topics;').then((result) => {
+    return db.query("SELECT * FROM topics;").then((result) => {
         return result.rows;
-        });
-}
-
+    });
+};
 
+const selectArticle = (id) => {
+    const query = "SELECT * FROM articles WHERE article_id = $1;";
+    let params = [id];
+    return db.query(query, params).then((result) => {
+        return result.rows;
```

Comment:
We don't want to be returning an array here. We want to be sending an individual object. How can we return just the object?

Reply:
We also need to add some custom error handling for 
Status 404, non existent ID, e.g. 0 or 9999.